connectors/sources/confluence.py

```diff
@@ -524,11 +554,19 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: dictionary containing meta-data of the content.
         """
+        await self.confluence_client.verify_spaces()
```

Comment:
As @artem-shelkovnikov already pointed I think we should remove verification logic from `get_docs`. At this point all verification/validations should be done already IMO. You could simply move that logic to `validate_config`, right? Then you can execute the declarative validation first and the "remote validation" afterwards (see [MySQL remote validation](https://github.com/elastic/connectors-python/blob/main/connectors/sources/mysql.py#L393) as an example)